### DISCUSSION TITLE
3° Info Augusto - 1º Tri Trabalho

### DISCUSSION THREAD

**Discussion by profmessias** [profmessias/curriculo#8] [Comentários]
Coloquem a turma e o nome completo de vocês com o link do seu currículo.

**Comment by synaradaniel** [comment]
Synara Daniel Vieira 3I01: https://github.com/synaradaniel/curriculo

**Comment by jhulia23** [comment]
Jhulia Leal Barbosa 3|01: https://github.com/jhulia23/Curriculo

**Comment by jana07yag** [comment]
Janaina Aparecida dos Santos 3°I01 https://github.com/jana07yag/curriculo

**Comment by Thalyanecarvalho** [comment]
Thalyane carvalho de jesus 3I01:https://github.com/Thalyanecarvalho/curriculo

**Comment by JAMILYSILCA** [comment]
jamily silva catarina 3I01:https://github.com/JAMILYSILCA/curricolo
